aries-site/src/data/structures/templates/templates.js

```diff
@@ -607,8 +607,8 @@ export const templates = [
     name: 'Banner Notifications',
```

Comment:
Should this page be called "Global Banner Notifications"? Since all the guidance is around global ones.

Reply:
good point I agree